src/foam/u2/view/TableCellFormatter.js

```diff
@@ -57,7 +57,7 @@ foam.CLASS({
         return foam.u2.view.FnFormatter.create({
           class: 'foam.u2.view.FnFormatter',
           f: function(value, obj, axiom) {
-            this.add(value);
+            this.add(obj[axiom.name + '$']);
```

Comment:
I noticed that properties don't change in the tableview when they change and this fixes it. Not sure if we'd want to do this. It'll cause lots of bindings.